internal/httprule/parse_test.go

```diff
@@ -294,6 +295,76 @@ func TestParseSegments(t *testing.T) {
 	}
 }
 
+func TestParse(t *testing.T) {
+	for _, spec := range []struct {
+		input       string
+		wantFields  []string
+		wantOpCodes []int
+		wantPool    []string
+		wantVerb    string
+	}{
+		{
+			input: "/v1/{name}:bla:baa",
+			wantFields: []string{
+				"name",
+			},
+			wantPool: []string{"v1", "name"},
+			wantVerb: "bla:baa",
+		},
+		{
+			input: "/v1/{name}:",
```

Comment:
I originally made this an error, but I'm not sure we can introduce an error for this now since it's been allowed up until now.

Reply:
This would be a weird path, but I don't see a problem with allowing it.